curriculum/challenges/english/25-front-end-development/workshop-loan-qualification-checker/66c8ba975ee7230e29f6c4ba.md

```diff
@@ -0,0 +1,77 @@
+---
+id: 66c8ba975ee7230e29f6c4ba
+title: Step 19
+challengeType: 1
+dashedName: step-19
+---
+
+# --description--
+
+If you change the `annualIncome` and `creditScore` variables to fall below the minimum required all the loans, you would see a message that looks like this:
+
+> Your credit score is 50 and your annual income is 3000. So, You qualify for the following loan(s):
+
+This means you need to account for a situation in which the applicant qualifies for no loans.
+
+To do this, create an `if` statement with the condition `"qualificationMessage === 'You qualify for the following loan(s):\n \n"`.
```

Comment:
same here.
we should update this to account for multiple correct answers for string concatenation.

Reply:
This just checks for that string in particular. It's not concatenating any variable.